packages/eslint-config-react-app/base.js

```diff
@@ -29,8 +29,18 @@ module.exports = {
   parserOptions: {
     ecmaVersion: 2018,
     sourceType: 'module',
-    ecmaFeatures: {
-      jsx: true,
+    requireConfigFile: false,
+    babelOptions: {
```

Comment:
Is it possible for us to use the `babelOptions.configFile` option to point to CRA's babel config in node_modules? That was the intended use case for adding the `babelOptions` configuration. If that doesn't work, what can we do to improve the API?

Reply:
The `eslint-config-react-app` can be used without CRA: https://github.com/facebook/create-react-app/pull/10761#discussion_r611295242. We could allow users to provide their own `babelConfig`, but we can not make it be a required option, otherwise it have to be shipped in a breaking change.

The idea of this PR is to try to stick with the old `babel-eslint@10` behaviour (parsing stage 1 proposals/react/jsx) so it is not a breaking change.

Reply:
I've updated this to reference our Babel config, which is now a peer dependency.